lib/bankaccount/account.rb

```diff
@@ -0,0 +1,37 @@
+module BankAccount
+
+	class Account
+
+		attr_accessor :id, :initial_balance, :account_balance
+
+		def initialize(id, initial_balance)
+			raise ArgumentError.new "Please enter a positive initial balance." unless initial_balance >= 0
+				@id = id
+				@initial_balance = initial_balance
+				@account_balance = initial_balance
+		end
+
+		def withdraw(amount)
```

Comment:
I like this method. One possibility for improvement might be to start by checking for the error condition first and only setting the variable to a new value if it passes the condition. It would save you a line or two.